File: assets/scripts/pingpongposition.py
import dengine.common as common
import dengine.scene as scene
import dengine.timer as timer
import math


def update(entity):
    current = timer.get_current()
    pingpong = math.sin(current / 1000.0)
    entity.transform.position.x = pingpong * 5.0

File: nsl/src/nsl-test/pingpongposition.c
#include <dengine/dengine.h>
#include <math.h>

int pingpongposition_update(Entity* entity)
{
    double current = dengineutils_timer_get_current();
    double pingpong = sin(current / 1000.0);
    entity->transform.position[0] = pingpong * 5.0;
    return 0;
}
